### ISSUE TITLE
Zmienna na wylewanie wiaderka na terenie wrogiej gildii

### ISSUE THREAD

**Issue by Ravis96** [FunnyGuilds/FunnyGuilds#1912]
smt (config)

**Comment by Kamilkime** [comment]
W sensie... Chcesz żeby dało się w configu zmienić czy można na terenie wrogiej gildii używać wiaderek?

**Comment by P3ridot** [comment]
Wygląda jakby była taka opcja https://github.com/FunnyGuilds/FunnyGuilds/blob/master/plugin/src/main/java/net/dzikoysk/funnyguilds/listener/region/BucketAction.java
Tylko bez możliwości rozróżnienia kto może wylewać

**Comment by Ravis96** [comment]
> Wygląda jakby była taka opcja https://github.com/FunnyGuilds/FunnyGuilds/blob/master/plugin/src/main/java/net/dzikoysk/funnyguilds/listener/region/BucketAction.java Tylko bez możliwości rozróżnienia kto może wylewać

Dokładnie. Wystarczyłoby dodać po prostu filtr do eventu i field do configu -- sprawa załatwiona.

**Comment by P3ridot** [comment]
Nie rozumiem, przecież dodając odpowiedni Material do już istniejącego fielda powinno działać
